torch/linalg/__init__.py

```diff
@@ -813,6 +813,102 @@
     https://www.netlib.org/lapack/lug/node128.html
 """)
 
+ldl_factor_ex = _add_docstr(_linalg.linalg_ldl_factor_ex, r"""
+linalg.ldl_factor_ex(A, *, upper=False, hermitian=False, check_errors=False, out=None) -> (Tensor, Tensor, Tensor)
+
+Computes the LDL decomposition of a Hermitian/symmetric indefinite matrix.
```

Comment:
```suggestion
Computes the LDL decomposition of a Hermitian/symmetric matrix.
```

It'd be good to have a mathematical definition of this function somewhere in the description, similar to that in SVD
https://pytorch.org/docs/stable/generated/torch.linalg.svd.html
Even better, this should be in a potential `linalg.ldl` function, and here we should refer to the docs of this other function.

Reply:
SVD is easy to describe because the return is exactly the matrices that define the decomposition. Here the decomposition is in compact form. We don't have this `linalg.ldl` function yet, so we shouldn't refer to it. This function does refer to `linalg.ldl_solve` which is created to work with the compact representation this function returns. I think it's rather difficult and unnecessary to describe the details of the representation. It's enough to mention that it's supposed to be used with `ldl_solve`.

Reply:
Is your suggestion to remove the "indefinite" word in this sentence?

Reply:
Re the definite, yes.

About that, sure thing. I think it would be good to follow a description as similar similar to that in https://pytorch.org/docs/master/generated/torch.linalg.lu_factor.html as possible for consistency.
If describing the format of the decomposition is too difficult, I'd say we put in its place a link to the relevant LAPACK page where it's described.